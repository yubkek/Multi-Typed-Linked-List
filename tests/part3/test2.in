NEW 3
1
c
32e2
NEW 4
a
b
character
324
INSERT 0 0 {1}
NEW 2
HDFGHD
{0}
