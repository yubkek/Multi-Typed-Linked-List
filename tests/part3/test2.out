List 0: 1 -> c -> 3200.00
List 1: a -> b -> character -> 324
